src/callbacks_step/analysis.jl

```diff
@@ -236,10 +250,15 @@ function (analysis_callback::AnalysisCallback)(integrator)
     mpi_println(" sim. time:      " * @sprintf("%10.8e", t) *
                 "               " *
                 " time/DOF/rhs!:  " * @sprintf("%10.8e s", runtime_relative))
+    mpi_println("                 " * "              " *
+                "               " *
+                " PID/walltime:   " * @sprintf("%10.8e s", pid_walltime))
     mpi_println(" #DOF:           " * @sprintf("% 14d", ndofs(semi)) *
+                "               " *
+                " PID/coretime:   " * @sprintf("%10.8e s", pid_coretime))
```

Comment:
```suggestion
                " PID(walltime):  " * @sprintf("%10.8e s", pid_walltime))
     mpi_println(" #DOF:           " * @sprintf("% 14d", ndofs(semi)) *
                "               " *
                " PID(coretime):  " * @sprintf("%10.8e s", pid_coretime))
```
I don't like the division operator here since it has a different meaning in the other PID (the one we have used so far).

(I'm using my phone so please check the indentation :sweat_smile:)

Reply:
Good point. Fixed in 2505ac28.